package.json

```diff
@@ -27,7 +27,7 @@
     "workspace-schematic": "nx workspace-schematic",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "nx": "nx",
+    "nx": "node --max_old_space_size=8000 ./node_modules/.bin/nx",
```

Comment:
do we need this? currently we dont have the power on ci

Reply:
it does not get executed on the ci only for local speed improvements